cli/lsp/language_server.rs

```diff
@@ -2126,13 +2126,16 @@ impl Inner {
     }
   }
 
-  async fn request_else(
+  async fn execute_command(
```

Comment:
Actually, this solution isn't acceptable, because it is a significant breaking change with all clients.  We can't merge this.  Clients use custom methods, not "workspace/executeCommand" that is a totally different LSP API.  Basically every client would break if we merged this. :sad: